src/devices/vesternet.ts

```diff
@@ -15,7 +42,8 @@ const definitions: Definition[] = [
         fromZigbee: extend.light_onoff_brightness().fromZigbee
             .concat([fz.electrical_measurement, fz.metering, fz.ignore_genOta]),
```

Comment:
I had to delete fz.level_config to pass the pipeline, it works but why?

Reply:
Because `fz.level_config` is available to all lights using `extend.light_onoff_brightness()` (duplicate converters are not allowed)

https://github.com/Koenkk/zigbee-herdsman-converters/blob/c3e653bf4a221b121afba26f41f4702bd071b298/src/lib/extend.ts#L27

Reply:
ok thx